.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    env:
+      CI: true
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install
+      run: npm ci
```

Comment:
Does this `npm ci` command cache the `node_modules` directory?

Reply:
Nope but it installs exactly what's in the lockfile so it's pretty fast (+ there are very few dependencies at the moment)

<img width="274" alt="Screenshot 2020-04-20 at 18 51 03" src="https://user-images.githubusercontent.com/6459679/79783059-f67c4280-8337-11ea-9ca5-d7af283334f8.png">

Reply:
Ah okay. Maybe caching is one to keep in mind if dependencies list grows in the future (I can't imagine it will too much).